rds-ca-2019-2015-root.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
As we've found out, Aurora serverless requires a different cert - (this one)[https://github.com/dwp/manage-mysql-user/blob/master/AmazonRootCA1.pem]. You can get rid from rds-ca-2019-2015-root.pem as k2hb doesn't need compatibility with any other RDS flavour.

Reply:
Fixed